src/dropdown/Dropdown.js

```diff
@@ -9,22 +10,16 @@ const themes = {
 };
 
 const PropTypes = {
-  children: React.PropTypes.oneOfType([
-    React.PropTypes.node,
-    React.PropTypes.element,
-    React.PropTypes.array
-  ]),
-  theme: React.PropTypes.oneOf(['semantic']),
-  valueLink: React.PropTypes.shape({
-    value: React.PropTypes.string,
-    requestChange: React.PropTypes.func
-  })
+  children: t.ReactNode,
+  theme: t.maybe(t.enums.of(['semantic'])),
+  valueLink: t.maybe(t.struct({
+    value: t.maybe(t.String),
+    requestChange: t.Function
+  }))
 };
-
+@props(PropTypes, { strict: false })
```

Comment:
yes, definitely. I was talking with @gabro about this issue two days ago: are `docgen` should identify `{ strict: false}` and also parse eventual comment describing **why** props are not strict.
PS: I'm going to add this comment when working on the *"add inline docs"* part of #204